components/components.css

```diff
@@ -0,0 +1,400 @@
+}
+
+.img-round {
+  width: 100%;
+  height: 100%;
+  border-radius: 50%;
+  object-fit: cover;
+}
+
+.img-rect-round {
+  width: 200px;
+  height: 200px;
+  border-radius: 50%;
+  object-fit: cover;
+}
+
+/* AVATAR */
+
+.avatar-sl {
+  height: 24px;
+  width: 24px;
+  position: relative;
+  display: inline-block;
+  user-select: none;
+}
+
+.avatar-lg {
+  height: 40px;
+  width: 40px;
+  position: relative;
+  display: inline-block;
+  user-select: none;
+}
+
+.avatar-xl {
+  height: 52px;
+  width: 52px;
+  position: relative;
+  display: inline-block;
+  user-select: none;
+}
+
+/* BADGES */
+
+.badge {
+  height: 10px;
+  width: 10px;
+  border-radius: 50%;
+  position: absolute;
+  z-index: 1;
+  right: 2%;
+  bottom: 2%;
+}
+
+.dot-green {
+  background-color: green;
+  box-shadow: 0px 0px 0px 2px #ffffff;
+}
+
+.dot-red {
+  background-color: red;
+  box-shadow: 0px 0px 0px 2px #ffffff;
+}
+
+.badge-text-green {
+  background: var(--success-green);
+  font-size: 12px;
+  color: white;
+  text-transform: uppercase;
+  font-weight: 500;
+  padding: 0 0.25rem;
+}
+
+.badge-text-yellow {
+  background: var(--warning-yellow);
+  font-size: 12px;
+  color: white;
+  text-transform: uppercase;
+  font-weight: 500;
+  padding: 0 0.25rem;
+}
+
+.badge-text-red {
+  background: var(--danger-red);
+  font-size: 12px;
+  color: white;
+  text-transform: uppercase;
+  font-weight: 500;
+  padding: 0 0.25rem;
+}
+
+.badge-text-blue {
+  background: var(--info-blue);
+  font-size: 12px;
+  color: white;
+  text-transform: uppercase;
+  font-weight: 500;
+  padding: 0 0.25rem;
+}
+
+/* ALERTS */
+
+.alert {
+  padding: 12px 16px;
+  display: flex;
+  align-items: center;
+  line-height: 22px;
+  border-radius: 4px;
+}
+
+.alert svg {
+  margin-right: 8px;
+}
+
+/* Error Alert */
+
+.error {
+  background-color: rgb(254, 215, 215);
+  color: #e53e3e;
+}
+
+.error svg {
+  fill: #e53e3e;
+}
+
+/* Success Alert */
+
+.success {
+  background-color: #c6f6d5;
+  color: #38a169;
+}
+
+.success svg {
+  fill: #38a169;
+}
+
+/* Warning Alert */
+
+.warning {
+  background-color: #feebc8;
+  color: #dd6b20;
+}
+
+.warning svg {
+  fill: #dd6b20;
+}
+
+/* Info Alert */
+
+.info {
+  background-color: #bee3f8;
+  color: #3182ce;
+}
+
+.info svg {
+  fill: #3182ce;
+}
+
+/* CARDS */
+
+/* Wisthlist Button */
+
+.wishlist-button {
+  display: inline-block;
+  position: absolute;
+  height: 24px;
+  width: 24px;
+  border-radius: 50%;
+  top: 6px;
+  right: 6px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  cursor: pointer;
+  background: rgba(255, 255, 255, 0.5);
+}
+
+.wishlist-button > svg {
+  fill: var(--gray-three);
+}
+
+/* Close Button */
+
+.close-button {
+  display: inline-block;
+  position: absolute;
+  height: 24px;
+  width: 24px;
+  border-radius: 50%;
+  top: 6px;
+  right: 6px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  cursor: pointer;
+  background: rgba(255, 255, 255, 0.5);
+}
```

Comment:
wishlist-button and add-button have common styles flex cursor: pointer etc. better if they are extracted to an independent class and reused

Reply:
Yes, I've improved the rules for card buttons👍